test/nuxeo-breadcrumb.test.js

```diff
@@ -99,7 +99,70 @@ suite('nuxeo-breadcrumb', () => {
     test('Should display the document title when a document is set and has a title', async () => {
       const title = breadcrumb.shadowRoot.querySelector('.current');
       expect(isElementVisible(title)).to.be.true;
-      expect(title.innerText).to.equal('my file');
+      expect(title.innerText).to.equal('my file (874fghd) ');
```

Comment:
Why do we need the extra space after the ")" ?
```suggestion
      expect(title.innerText).to.equal('my file (874fghd)');
```